### PR TITLE
Adding job poster GH handle field to form

### PR DESCRIPTION
**Pull request by ei8fdb**
As explained in #384.

### CODE REVIEW

**Comment by eupiteco** [comment]
As I just mentioned in #384 (sorry for the mess and double post, delete wich one is necessary) It would be nice to add these fields' values to the posted job at the job boards (osd.net/jobs). You can check any docs nedded [here](https://shopify.github.io/liquid/), and I can help with that too (:

**Comment by ei8fdb** [comment]
> As I just mentioned in #384 (sorry for the mess and double post, delete wich one is necessary) It 
> would be nice to add these fields' values to the posted job at the job boards (osd.net/jobs). You can > check any docs nedded [here](https://shopify.github.io/liquid/), and I can help with that too (:

Hmm, I *think* the extra fields will be posted to the jobs board automatically. I'm not 100%. I'll ask @belenbarrospena as she wrote the jobs board form.

Either way, the GH handle is more useful really for the PR reviewers (i.e. us people here!), in case we need to ask the poster a question.

File: job-form.html [hunk 1]
```diff
@@ -135,6 +135,15 @@ <h1>Submit a Job</h1>
 			name="fields[how_to_apply]">
 		  <small>Markdown is okay</small>
 		</div>
+    <div class="form-group">
```
Comment (evalica) [inline job-form.html]: the indentation is not consistent :)

File: job-form.html [hunk 2]
```diff
@@ -135,6 +135,15 @@ <h1>Submit a Job</h1>
...
+      <p class="help-block"></p>
+      <input class="form-control"
+      id="how_to_apply"
+      rows="3"
+      required
+      placeholder="GitHub @username (optional)"
+      name="fields[how_to_apply]">
```
Comment (eupiteco) [inline job-form.html]:
```suggestion
      name="fields[how_to_apply]" />
```
Missing that slash to close the `<input />` tag

**Comment by ei8fdb** [comment]
I *think* I've resolved the 2 issues mentioned above. /crosses fingers.

File: job-form.html [hunk 3]
```diff
@@ -135,6 +135,15 @@ <h1>Submit a Job</h1>
 			name="fields[how_to_apply]">
 		  <small>Markdown is okay</small>
 		</div>
+		<div class="form-group">
+		      <p class="help-block"></p>
+		      <input class="form-control"
+		      id="how_to_apply"
```
Comment (jancborchardt) [inline job-form.html]: The id `how_to_apply` is duplicated from above. Could you change it to `github_handle`?

File: job-form.html [hunk 4]
```diff
@@ -135,6 +135,15 @@ <h1>Submit a Job</h1>
...
+		      rows="3"
+		      required
```
Comment (jancborchardt) [inline job-form.html]: Since it’s optional, this `required` needs to be removed. :)

File: job-form.html [hunk 5]
```diff
@@ -135,6 +135,15 @@ <h1>Submit a Job</h1>
...
+		      placeholder="GitHub @username (optional)"
+		      name="fields[how_to_apply]"/>
```
Comment (jancborchardt) [inline job-form.html]: Same here, `fields[github_handle]`

File: job-form.html (hunk 3 above)
Comment (jancborchardt) [reply job-form.html]:
```suggestion
		      id="github_handle"
```

File: job-form.html (hunk 4 above)
Comment (jancborchardt) [reply job-form.html]:
```suggestion
```

File: job-form.html (hunk 5 above)
Comment (jancborchardt) [reply job-form.html]:
```suggestion
		      name="fields[github_handle]"/>
```

File: job-form.html [hunk 6]
```diff
@@ -135,6 +135,14 @@ <h1>Submit a Job</h1>
 			name="fields[how_to_apply]">
 		  <small>Markdown is okay</small>
 		</div>
+		<div class="form-group">
+		      <p class="help-block"></p>
+		      <input class="form-control"
+		      id="github_username"
```
[the remark this answers is not in the document]
Comment (jancborchardt) [reply job-form.html]:
```suggestion
		      id="github_handle"
```

**Comment by RDIL** [comment]
Works for me

**Pull request merged**